subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1953,6 +1979,14 @@ void ull_conn_tx_ack(uint16_t handle, memq_link_t *link, struct node_tx *tx)
 		pdu_tx->ll_id = PDU_DATA_LLID_RESV;
 	} else {
 		LL_ASSERT(handle != LLL_HANDLE_INVALID);
+
```

Comment:
Missing "if defined( CONFIG_BT_LL_SW_LLCP_LEGACY)" ?

Reply:
@erbr-ot Do I need the conditional compilation as non of the CI tests are complaining?

Reply:
Well, the rx_hold-functionality is under CONFIG_BT_LL_SW_LLCP_LEGACY is it not? So I'd expect the usage to be as well?

Reply:
I guess no builds are made with refactored LLCP AND CONFIG_BT_CTLR_RX_ENQUEUE_HOLD?

Reply:
@kruithofa can confirm, the bsim LL conformance tests run the refactored LLCP, but I do not think CONFIG_BT_CTLR_RX_ENQUEUE_HOLD would be enabled?

Reply:
@erbr-ot I have added the depends on CONFIG_BT_LL_SW_LLCP_LEGACY